### PR TITLE
Containerization

### PR DESCRIPTION
**Pull request by fernandoacorreia**
WIP

### CODE REVIEW

File: _docs/upgrading/containerization.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+When `domino.is_containerized` is `'true'`, its value must be an empty string.
+
+### Legacy properties
+
+These properties are not supported anymore:
+
+**domino.artifact.url**
+
+Legacy property for the URL of the Domino application tarball. It has been replaced first by `domino.artifacts.server` and then by `domino.artifacts.executor` and `domino.artifacts.nucleus`. It must not be used anymore.
+
```
Comment (chrismyang) [inline _docs/upgrading/containerization.md]: In practice, i don't think i've ever actually seen anything use `domino.artifacts.server` only `domino.artifacts.dir`

File: _docs/upgrading/containerization.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+  images:
+    executor: quay.io/domino/executor:releases-2.4.0.e675a05c84d9d45c5076f2e348f001725aa296ba
+    nucleus: quay.io/domino/nucleus:releases-2.4.0.e675a05c84d9d45c5076f2e348f001725aa296ba
+  is_containerized: 'true'
+mongodb:
+  image: quay.io/domino/mongodb:releases-2.4.0.e675a05c84d9d45c5076f2e348f001725aa296ba
+  is_containerized: 'true'
+```
+
+## Operations
```
Comment (chrismyang) [inline _docs/upgrading/containerization.md]: mostly leaving this as a note for myself, bug I think we should move this into some separate troubleshooting/operations document (and link to there).  this advice is timeless, where as the rest of this content is really only relevant for a 2.4 upgrade.

File: _docs/upgrading/containerization.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+```
+$ docker exec -it container-name bash
+```
+
+Exiting the shell will not stop the container.
+
+## Known Limitations
+
+* Changing the `Name` tag of executor EC2 instances is not supported.
+* Admin / Advanced / Docker form is not supported.
```
Comment (chrismyang) [inline _docs/upgrading/containerization.md]: The fact that you wrote "limitation is expected to be lifted..." for all the other items suggests that you do not intend to ever add this feature back.  Is that your intent?

File: _docs/upgrading/containerization.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
+---
+title: Containerization
+category: Upgrading
+order: 1
```
Comment (chrismyang) [inline _docs/upgrading/containerization.md]: Probably shouldn't be the first item in the list (which is what this `order: 1` means...)

**Comment by chrismyang** [comment]
overall looks fine.  left a few comments that are probably larger discussions

File: _docs/upgrading/containerization.md (hunk 1 above)
Comment (fernandoacorreia) [reply _docs/upgrading/containerization.md]: Sound plausible, but as the code still mentions these old settings I want to document that they're not supported anymore.

File: _docs/upgrading/containerization.md (hunk 2 above)
Comment (fernandoacorreia) [reply _docs/upgrading/containerization.md]: Yes, we'll need to restructure the pages (and find a way to link to different versions).

File: _docs/upgrading/containerization.md (hunk 3 above)
Comment (fernandoacorreia) [reply _docs/upgrading/containerization.md]: For the other items, we already know that we'll add them. For this one, it's still to be discussed. There's a ticket open.

File: _docs/upgrading/containerization.md (hunk 4 above)
Comment (fernandoacorreia) [reply _docs/upgrading/containerization.md]: Yeah I just wasn't sure if your intent is that as new versions are added, the order in the old pages will have to be manually edited. Doesn't matter at the moment.

**Pull request merged**